README.md

```diff
@@ -34,11 +31,15 @@ For more information in the source, here are some pointers:
 - [boot/bootutil](https://github.com/runtimeco/mcuboot/tree/master/boot/bootutil): The core of the bootloader itself.
 - [boot/boot\_serial](https://github.com/runtimeco/mcuboot/tree/master/boot/boot_serial): Support for serial upgrade within the bootloader itself.
 - [boot/zephyr](https://github.com/runtimeco/mcuboot/tree/master/boot/zephyr): Port of the bootloader to Zephyr
+- [boot/mynewt](https://github.com/runtimeco/mcuboot/tree/master/boot/mynewt): Mynewt bootloader app
 - [imgtool](https://github.com/runtimeco/mcuboot/tree/master/imgtool): A tool to securely sign firmware images for booting by mcuboot.
 - [sim](https://github.com/runtimeco/mcuboot/tree/master/sim): A bootloader simulator for testing and regression
 
-## Joining 
+## Joining
 
-Developers welcome!  To join in the discussion, please join the developer mailing list: 
+Developers welcome!
 
-http://lists.runtime.co/mailman/listinfo/dev-mcuboot_lists.runtime.co
+* Our developer mailing list:
+  http://lists.runtime.co/mailman/listinfo/dev-mcuboot_lists.runtime.co
+* Our Slack channel: https://runtimeco.slack.com/
```

Comment:
I believe Slack requires an invitation to join. Not sure if it would be obvious to people that they need to send email and ask for an invite.